### PR TITLE
Add servicenow extension

### PR DESCRIPTION
**Pull request by robertoalvarezalonso**
## Description
The extension enables users to easily search and explore content across their ServiceNow instances with different user profiles. Users can perform quick searches, manage instance profiles, and open specific instances or ServiceNow records in their browser.

## Screencast
🎥 [Video](https://swissflowit.sharepoint.com/:v:/s/founders/EWsS-7rLwTpMiXXbKwZCCWkBhVimrFsExw87n3yxzwj87w?e=udtEvi)

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by pernielsentikaer — COMMENTED** [review]
I am not very familiar with ServiceNow. Could you create a screencast demonstrating a few of the views?

File: extensions/servicenow/src/components/SearchList.tsx [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+                          mutateInstances();
+                        }}
+                        target={selectedInstance && <SearchResults searchTerm={item.search_term} />}
+                        title={`Search for "${item.search_term}"`}
+                        icon={Icon.MagnifyingGlass}
+                      />
+                      <Actions mutate={mutate} />
+                      <List.Dropdown.Section title="Term">
+                        <Action
+                          title="Remove from History"
```
Comment (pernielsentikaer) [inline extensions/servicenow/src/components/SearchList.tsx:243]:
```suggestion
                          title="Remove from History"
                          shortcut={Keyboard.Shortcut.Common.Remove}
```

File: extensions/servicenow/src/components/SearchList.tsx [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+                          icon={Icon.XMarkCircle}
+                          style={Action.Style.Destructive}
+                          onAction={() => removeItemFromHistory(item)}
+                        />
+                        <Action
+                          title="Clear All Items from History"
+                          shortcut={{ modifiers: ["cmd"], key: "backspace" }}
```
Comment (pernielsentikaer) [inline extensions/servicenow/src/components/SearchList.tsx:250]:
```suggestion
                          shortcut={Keyboard.Shortcut.Common.RemoveAll}
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
I only found a few shortcuts missing. Besides that, it looks really good - awesome! 🙂

**Comment by robertoalvarezalonso** [comment]
@pernielsentikaer thanks a lot! I've applied your changes and took advantage to review all the actions, adding Keyboard.Shortcut.Common where it made sense.

**Comment by pernielsentikaer** [comment]
Awesome 🔥 Approved ✅

**Pull request merged**